solidity/contracts/VerifierModule.sol

```diff
@@ -0,0 +1,279 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.19;
+
+import {IStorageMirror} from 'interfaces/IStorageMirror.sol';
+import {MerklePatriciaProofVerifier} from 'libraries/MerklePatriciaProofVerifier.sol';
+import {StateVerifier} from 'libraries/StateVerifier.sol';
+import {RLPReader} from 'solidity-rlp/contracts/RLPReader.sol';
+import {IStorageMirrorRootRegistry} from 'interfaces/IStorageMirrorRootRegistry.sol';
+import {IBlockHeaderOracle} from 'interfaces/IBlockHeaderOracle.sol';
+import {IVerifierModule} from 'interfaces/IVerifierModule.sol';
+import {ISafe} from 'interfaces/ISafe.sol';
+import {Enum} from 'safe-contracts/common/Enum.sol';
```

Comment:
Let's try to follow an import order with external import at top and under internal imports